docs/_docs/1_6_flink_quick_start_guide.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Quick-Start Guide"
+permalink: /docs/flink-quick-start-guide.html
+toc: true
+last_modified_at: 2020-03-16T11:40:57+08:00
+---
+
+This guide provides a quick peek at Hudi's capabilities using flink SQL client. Using flink SQL, we will walk through 
+code snippets that allows you to insert and update a Hudi table of default table type: 
+[Copy on Write](/docs/concepts.html#copy-on-write-table) and [Merge On Read](/docs/concepts.html#merge-on-read-table). 
+After each write operation we will also show how to read the data snapshot (incrementally read is already on the roadmap).
+
+## Setup
+
+We use the [Flink Sql Client](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/sqlClient.html) because it's a good
+quick start tool for SQL users.
+
+### Step.1 download flink jar
+Hudi works with Flink-1.11.x version. You can follow instructions [here](https://flink.apache.org/downloads.html) for setting up flink.
+The hudi-flink-bundle jar is archived with scala 2.11, so it’s recommended to use flink 1.11 bundled with scala 2.11.
+
+### Step.2 start flink cluster
```

Comment:
Can we share some details about how to use the flink docker to simplify the usage?

Reply:
What do you mean for flink docker ?

Reply:
I mean [Flink docker setup](https://ci.apache.org/projects/flink/flink-docs-stable/deployment/resource-providers/standalone/docker.html)

Reply:
We do not need docker here, a local cluster is simple enough.

Reply:
We can add more docker demo about Flink on this page: http://hudi.apache.org/docs/docker_demo.html